app/helpers/wikis_helper.rb

```diff
@@ -1,2 +1,5 @@
 module WikisHelper
+  def seeWiki(user, wiki)
```

Comment:
I would call this something like `can_see_wiki?`

Also, it's convention to use snake case not camel case in Ruby.

https://en.wikipedia.org/wiki/Snake_case
https://en.wikipedia.org/wiki/Camel_case